packages/gatsby-theme-docs/src/components/video-player.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+import { customProperties } from '@commercetools-uikit/design-system';
+import styled from '@emotion/styled';
+
+const VideoPlaceholder = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  aspect-ratio: 16 / 9;
```

Comment:
It's pretty good not but still has a little bit of shift in the first example video because 16:9 does not include the control bar of the video player (30px).   
you can add a bottom padding of 30px, that will make it fit. 
```suggestion
  aspect-ratio: 16 / 9;
  padding-bottom: 30px;
```

Alternatively a youtube-ish layout with the control bar fading in only on hover could be done but I think we should not overengineer the interaction pattern.

Reply:
Checking the current CSS applied to the controls turns out the height is set to `3em`
I set the same to the bottom-padding of the placeholder.

<img width="896" alt="Screenshot 2022-10-26 at 14 36 59" src="https://user-images.githubusercontent.com/106664971/198028060-751a05cf-7048-497e-8332-708232153dc3.png">

BTW, I still see layout shift. It seems like it happens within the videojs implementation. As soon as the client component loads, just before it shows the poster, I can see the black area becoming smaller, and just after that jumping back to the normal size.
I'll keep looking into it although the overall effect is now much better 👍

Reply:
Thanks.  It's a bit frustrating the the videjs player has layout shift in itself while loading (it seems it's first reserving the space for the video and only then adding the control bars).  

So that remaining bit is out of our control, at least in the current scope to not overdo it before we have actual content.